infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -252,16 +252,66 @@ locals {
         "/api/1.0/tasks?project={ANY_PROJECT_GID}",
         "/api/1.0/tasks/{ANY_TASK_GID}",
         "/api/1.0/tasks/{ANY_TASK_GID}/stories",
-      ],
-      secured_variables : [
-        { name : "ACCESS_TOKEN", writable : false },
-      ],
+      ]
       external_token_todo : <<EOT
   1. Create a [Service Account User + token](https://asana.com/guide/help/premium/service-accounts)
     or a sufficiently [Personal Access Token](https://developers.asana.com/docs/personal-access-token)
     for a sufficiently privileged user (who can see all the workspaces/teams/projects/tasks you wish to
     import to Worklytics via this connection).
   2. Update the content of PSOXY_ASANA_ACCESS_TOKEN variable with the previous token value obtained
+EOT
+    }
+    salesforce = {
+      source_kind : "salesforce",
+      worklytics_connector_id : "salesforce-psoxy"
+      display_name : "Salesforce"
+      identifier_scope_id : "salesforce"
+      worklytics_connector_name : "Salesforce via Psoxy"
+      target_host : var.salesforce_domain
+      source_auth_strategy : "oauth2_refresh_token"
+      environment_variables : {
+        GRANT_TYPE : "client_credentials"
+        CREDENTIALS_FLOW : "client_secret"
```

Comment:
This is to define how client credentials will work (based on client secret or based on JWT assertion)